src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocsummarylocation/InputJavadocSummaryLocationDefault.java

```diff
@@ -0,0 +1,25 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocsummarylocation;
+
+/**
+ * config: default
```

Comment:
for future ...
We should probably place here all properties with their default values,  it will be much easier to read, no need to switch context to search what is default.